tasks/metrics/mutant.rake

```diff
@@ -19,6 +19,13 @@ namespace :metrics do
       %W[--ignore #{matcher}]
     end
 
+    jobs =
+      if ENV.fetch('DEVTOOLS_SELF', false).eql?('1') && ENV.key?('CIRCLECI')
```

Comment:
(I'll remove the `DEVTOOLS_SELF` check unless you agree with my statement above @mbj)